### ISSUE TITLE
ZigBee device manufacturer name issue

### ISSUE THREAD

**Issue by lucasvdh** [athombv/apps-sdk-issues#89]
I wrote a PR for the new IKEA blinds and the open/close remote that comes along with it.
https://github.com/athombv/com.ikea.tradfri/pull/74

The new driver I wrote for the blinds works just fine, but I'm experiencing some strange issues when trying to get the remote working.

I've added the remote as a generic ZigBee device, the config I got from it was as follows:
```json
{
  "manufacturerName": "IKEA of Sweden",
  "productId": [
    "TRADFRI open/close remote"
  ],
  "deviceId": 515,
  "profileId": [
    260
  ]
}
```

## The issue
When I configure a driver with this zigbee config and I try to add the remote, Homey doesn't map the device to this driver and adds it as an unknown zigbee device. 

When I tried to figure out why it didn't match the remote to the driver I wrote for it, I noticed that the `manufacturerName` of the unknown device had a different value than before. It had changed from `"IKEA of Sweden"` to `"KE"`. 

If I change the zigbee config of the driver so that the `manufacturerName` is set as `"KE"` and add the remote again, it again doesn't match the device to the driver and the `manufacturerName` has reverted back to `"IKEA of Sweden"`.

You can look at the currently open pull request if you want more info on how the driver is configured.

**Comment by lucasvdh** [comment]
Further investigation reveals that the `manuName` has a value of `"\u0002KE"` instead of `"KE"`.

The device as listed in the zigbee state:
```json
{
  "type": "EndDevice",
  "ieeeAddr": "0x000d6ffffee3f573",
  "nwkAddr": 13029,
  "manufId": 4476,
  "manufName": "\u0002KE",
  "powerSource": "Battery",
  "modelId": "TRADFRI open/close remote",
  "epList": [
    1
  ],
  "status": "offline",
  "joinTime": 1566652758,
  "name": "Open/sluit afstandsbediening",
  "profileId": "260",
  "deviceId": "515",
  "manufacturerName": "KE",
  "productId": "TRADFRI open/close remote"
}
```

**Comment by lucasvdh** [comment]
Has there been any progress and is there anything I can do to help?

**Comment by henrikrox** [comment]
@jeroenvollenbrock @WeeJeWel

**Comment by jeroenvollenbrock** [comment]
We are aware of the issue and are working on a solution, but cannot give any time estimates.

**Comment by lucasvdh** [comment]
Any news?

**Comment by lucasvdh** [comment]
Could you maybe at least tell me if this will be fixed before v3 or if we're going to have to wait until the new version is released?

**Comment by pellz0r** [comment]
Still no news?

**Comment by lucasvdh** [comment]
We'll just have to be patient I guess.. 

@pellz0r I think I figured out a way to get the buttons working until this is fixed though. Just create a driver for both manufacturer names :sweat_smile: If I manage to get them working I'll prob post something to the [ikea app repo](https://github.com/athombv/com.ikea.tradfri)

**Comment by lucasvdh** [comment]
> Could you maybe at least tell me if this will be fixed before v3 or if we're going to have to wait until the new version is released?

v4?

**Comment by klim123123** [comment]
Still no update for TRADFRI open/close remote?
I just tried it in 4.2.0. Doesn't work.

**Comment by klim123123** [comment]
Just tried to add TRADFRI open/close remote using  lucasvdh/com.ikea.tradfri, 
getting message: No compatible Homey App has been found for this device. It has been added as Basic Zigbee device.

**Comment by lucasvdh** [comment]
@klim123123 I think this requires the ZigBee rewrite which Athom has been working on for a while.

**Comment by RobinBol** [comment]
Hi all, we have recently released Homey v5.0.0 to Experimental which supports com.ikea.tradfri@1.7.3 (currently available in test: https://homey.app/nl-nl/app/com.ikea.tradfri/IKEA-Tr%C3%A5dfri/test/). This combination allows pairing an open/close remote. We did not experience any of the issues described here, so these are probably fixed in the Zigbee engine update. I'll close this issue for now, please let me known if the issue is not resolved for you.

**Comment by klim123123** [comment]
Resolved! Thank you! I just tested 'Up/Down Remote Control' on 5.0.0-rc.18. It's just works!
